runtime/src/runtime/policy.rs

```diff
@@ -291,6 +307,16 @@ pub mod policy_constants {
     #[cfg(feature = "short-precommit")]
     pub const PRE_COMMIT_CHALLENGE_DELAY: ChainEpoch = 10;
 
+    // Maximum number of epochs within which to fetch a valid seal randomness from the chain for
+    // a non-interactive PoRep proof. This balances the need to tie the seal to a particular chain with
+    // but makes allowance for service providers to offer pre-sealed sectors within a larger window of
+    // time.
+    pub const MAX_PROVE_COMMIT_NI_LOOKBACK: ChainEpoch = 180 * EPOCHS_IN_DAY + CHAIN_FINALITY;
```

Comment:
```suggestion
    pub const MAX_PROVE_COMMIT_NI_LOOKBACK: ChainEpoch = 180 * EPOCHS_IN_DAY;
```

finality isn't in the spec and we clarified (on Slack I think) that it wasn't needed on this even though it would make it consistent with the others